content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -733,6 +734,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
   This feature gate will never graduate to stable.
 - `TopologyManagerPolicyOptions`: Allow fine-tuning of topology manager policies,
 - `UserNamespacesStatelessPodsSupport`: Enable user namespace support for stateless Pods.
+- `UnknownVersionInteroperabilityProxy`: Proxy resource requests to the correct peer kube-apiserver when
```

Comment:
This would collate earlier than `UserNamespacesStatelessPodsSupport`. Please fix the ordering.